support-frontend/assets/components/ticker/tickerContainer.tsx

```diff
@@ -0,0 +1,68 @@
+import { useEffect, useState } from 'react';
+import { fetchJson } from 'helpers/async/fetch';
+import type { TickerProps } from './ticker';
+import type {
+	TickerConfigData,
+	TickerCopy,
+	TickerCountType,
+	TickerEndType,
+} from './types';
+
+function getInitialTickerValues(
+	tickerCountType: TickerCountType,
+): Promise<TickerConfigData> {
+	return fetchJson<TickerConfigData>(
+		tickerCountType === 'money' ? '/ticker.json' : '/supporters-ticker.json',
```

Comment:
@i-hardy the data for a ticker can be retrieved from the support site route `/ticker/${tickerId}.json`. The `tickerId` must match the corresponding key from the ticker config from S3. This config is defined as JSON in S3 in the membership-private bucket, under `/{STAGE}/ticker.conf.json`. 

The `tickerId` for the US campaign should be "US". So the route to retrieve the data for the US will be `/ticker/US.json`. This route works on PROD and CODE, and will return the appropriate data for each environment. Each environments data is written to S3  in the contributions-ticker bucket, under `/{STAGE}/{TickerID}.json`. 

The routes above can be used whether the ticker is reporting on a money target OR a supporter target. 

For DEV if working on a ticker with a money target you can use the dummy route `/ticker.json`. If it's a ticker with a supporter target you can use the dummy route `/supporters-ticker.json`, both these routes contain dummy data that can be manipulated during dev.

This repo contains all this info: https://github.com/guardian/contributions-ticker-calculator